internal/compiler/passes/resolving.rs

```diff
@@ -1618,6 +1618,11 @@ fn resolve_two_way_bindings(
                                     } else {
                                         diag.push_error("Cannot link input property".into(), &node);
                                     }
+                                } else if rhs_lookup.property_visibility
+                                    == PropertyVisibility::InOut
+                                {
+                                    diag.push_warning("Link input property to input output property is deprecated".into(), &node);
```

Comment:
```suggestion
                                    diag.push_warning("Linking input properties to input output properties is deprecated".into(), &node);
```

(either that or singular with an article)